plugins/chat/assets/stylesheets/mobile/chat-user-threads.scss

```diff
@@ -1,10 +1,7 @@
 .c-user-thread {
   display: grid;
-  grid-template-areas:
-    "avatar category timestamp"
-    "avatar title indicator"
-    "avatar excerpt excerpt";
-  grid-template-columns: auto 1fr auto;
+  grid-template-areas: "avatar details";
+  grid-template-columns: auto 1fr;
```

Comment:
Do we even need these grid template areas here @chapoi ? That feels super overkill for what we are doing here and they are such a pain to read

Reply:
I had to make it a grid within a grid to get it to work with the newly wrapped `<a>` tag. I'm not completely sold on the grid layout approach (and why it's even needed) but I've adapted this to get it to work while retaining the grid template areas.

Reply:
Usually I don't use grid unless I have to for either desktop/mobile conversion, or for alignments. I'll take a look but it's most likely not overkill.

Reply:
Ah but I see now what you mean: there is a new grid lower in the hierarchy. Yes, than the previous grid is no longer needed and can be just flexed.